### ISSUE TITLE
import external svg

### ISSUE THREAD

**Issue by flip111** [terezka/lines#15]
can lines import an external svg like d3.js ?

https://bl.ocks.org/mbostock/1014829

**Comment by terezka** [comment]
No, not right now and probably not in the first few releases. A cool feature though!
